components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.html

```diff
@@ -0,0 +1,42 @@
+<label>{{ objectNounPlural | capitalize }}</label>
```

Comment:
This was copied verbatim from the old projects-dropdown component, then just replaced "projects" with `objectNounPlural` to genericize it.